src/StoreKeeper/WooCommerce/B2C/Commands/WpCliCommandRunner.php

```diff
@@ -31,6 +32,9 @@ private function registerCommand($name)
         \WP_CLI::add_command(
             self::command_prefix.' '.$name,
             function (array $arguments, array $assoc_arguments) use ($name) {
+                if (isset($assoc_arguments[self::ALLOW_SPAWN])) {
+                    $this->shouldSpawnSubProcess = true;
```

Comment:
Maybe we can just do it automatically instead of relaying on `self::ALLOW_SPAWN`? Something like this:

```suggestion

                    try {
                        $p = new Process('whoami');
                        $p->run();
                        $this->shouldSpawnSubProcess = true;
                    } catch(\Throwable $e){
                         $this->shouldSpawnSubProcess = false;
                         $logger->logAndEchoWarning($e);
                    }                    
```

Reply:
Added `isSubProcessesAvailable` method for testing subprocess.